Bottom: 6e79acc95e5cd324e988850c7bb7a5ba4b71e9ee
Top:    fe00ff19dcd6f47ab3550b53b4d712aca6fb4a23
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 14:38:55 +0100

Refresh of generate-pcb-menu-res-and-gpcb

---

diff --git a/src/Makefile.am b/src/Makefile.am
index f57483c..75c6886 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -246,7 +246,7 @@ pcb-menu.res : pcb-menu.res.in
 pcb-menu.h : pcb-menu.res
 	echo '/* AUTOMATICALLY GENERATED FROM pcb-menu.res DO NOT EDIT */' > $@
 	echo 'const char *pcb_menu_default[] = {' >> $@
-	sed 's/\\/\\\\/g; s/"/\\"/g; s/^/"/; s/$$/",/' < ${srcdir}/pcb-menu.res >> $@
+	sed 's/\\/\\\\/g; s/"/\\"/g; s/^/"/; s/$$/",/' < pcb-menu.res >> $@
 	echo '0};' >> $@
 
 gpcb-menu.res : gpcb-menu.res.in
@@ -256,7 +256,7 @@ gpcb-menu.res : gpcb-menu.res.in
 gpcb-menu.h : gpcb-menu.res
 	echo '/* AUTOMATICALLY GENERATED FROM gpcb-menu.res DO NOT EDIT */' > $@
 	echo 'const char *gpcb_menu_default[] = {' >> $@
-	sed 's/\\/\\\\/g; s/"/\\"/g; s/^/"/; s/$$/",/' < ${srcdir}/gpcb-menu.res >> $@
+	sed 's/\\/\\\\/g; s/"/\\"/g; s/^/"/; s/$$/",/' < gpcb-menu.res >> $@
 	echo '0};' >> $@
 
 hid/common/hidlist.h : Makefile
